borg/archiver.py

```diff
@@ -159,6 +160,36 @@ def do_change_passphrase(self, args, repository, manifest, key):
         key.change_passphrase()
         return EXIT_SUCCESS
 
+    @with_repository(lock=False, exclusive=False, manifest=False, cache=False)
+    def do_key_export(self, args, repository):
+        """Export the repository key for backup"""
+        manager = KeyManager(repository)
+        manager.load_keyblob()
+        if args.paper:
+            manager.export_paperkey(args.path)
+        else:
+            if not args.path:
+                self.print_error("output file to export key to expected")
+                return EXIT_ERROR
```

Comment:
as you also need args.path in line 169: move this check to above the if/else block?